### DISCUSSION TITLE
Comparison with HAR

### DISCUSSION THREAD

**Discussion by abitrolly** [vcr/vcr#853] [Q&A]
How similar VCR format to HAR? Is it possible to convert between HAR and VCR without losing any info?

**Comment by krainboltgreene** [comment]
Honestly great question. I think if I had to rebuild VCR from the ground up I would use a HAR file. That said, from VCR's perspective the format largely doesn't matter. We still have to do the work of parsing and then instructing the http mocking library on what to do.

**Comment by abitrolly** [comment]
Yes, I am having hard time trying to compare headers and redirects from browser session with my script that's using https://github.com/rest-client/rest-client Something is missing in scripted login workflow and I can not really figure out what.

**Comment by abitrolly** [comment]
If HAR and VCR are both JSON, then maybe side by side comparison of the fields would do the trick.
